docs/Compile.rst

```diff
@@ -121,7 +156,9 @@ DF, which causes DF to use your system libstdc++ instead::
 Note that distributing binaries compiled with newer GCC versions requires end-
 users to delete libstdc++ themselves and have a libstdc++ on their system from
 the same GCC version or newer. For this reason, distributing anything compiled
-with GCC versions newer than 4.8 is discouraged.
```

Comment:
OK great.  I'm pretty sure what I have covers that already, but let me know if it isn't clear or needs more explanation.

Or rather, what I have basically says "for now only 4.5 for distribution, anything >=4.5 for personal"

So I could clarify that actually 4.6, 4.7 and 4.8 might be OK for distribution, but I'm not sure that's necessary given distribution only needs to be available for a few people, all of whom have 4.5 available anyway.